pkg/controller/pxc/tls.go

```diff
@@ -57,14 +57,46 @@ func (r *ReconcilePerconaXtraDBCluster) createSSLByCertManager(cr *api.PerconaXt
 	ownerReferences := []metav1.OwnerReference{owner}
 
 	issuerName := cr.Name + "-pxc-ca"
+	selfSignedIssuerName := cr.Name + "-pxc-selfsigned"
```

Comment:
@hors we need to come up with better names for two issuers and two certificates.
also, we must check how these changes affect clusters during update from 1.8.0 to 1.9.0